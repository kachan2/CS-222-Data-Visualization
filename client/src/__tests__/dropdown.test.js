import React from "react";
import ShallowRenderer from 'react-shallow-renderer';

import DropDown from "../buttons/dropdown";

it('renders correctly react-test-renderer', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<DropDown />);
    const result = renderer.getRenderOutput();
  
    expect(result).toMatchSnapshot();
});
  