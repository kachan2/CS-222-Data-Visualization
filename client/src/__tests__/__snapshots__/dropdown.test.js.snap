// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<Dropdown
  menu={
    Array [
      <button
        onClick={[Function]}
      >
        Menu 1
      </button>,
      <button
        onClick={[Function]}
      >
        Menu 2
      </button>,
    ]
  }
  trigger={
    <button>
      Dropdown
    </button>
  }
/>
`;
